openlibrary/templates/admin/people/view.html

```diff
@@ -214,9 +213,24 @@ <h1><a class="plain" href="/admin/people">People</a> / $person.username</h1>
                 <a class="tag $cond(person.has_tag(t), 'active')">$t</a>
         </td>
     </tr>
+    <tr>
+        <td>$_("Anonymize Account:")</td>
+        <td>
+            <form method="POST" id="anonymize-form">
+                <button type="submit" name="action" value="anonymize_account" style="float: right;" onclick="confirmAnonymize(event)">$_("Anonymize Account")</button>
+            </form>
+        </td>
+    </tr>
     </tbody>
   </table>
 
+<script>
+    function confirmAnonymize(event) {
+        if(!confirm('Really anonymize this patron? This will delete the patron\'s profile page and booknotes, and anonymize the patron\'s reading log, reviews, and star ratings.')) {
+            event.preventDefault()
+        }
+    }
+</script>
```

Comment:
If we're happy with the confirmation dialog approach, I can move this inline JS to a `.js` file.